service/worker/deletenamespace/reclaimresources/activities.go

```diff
@@ -85,12 +81,7 @@ func NewLocalActivities(
 }
 
 func (a *LocalActivities) IsAdvancedVisibilityActivity(_ context.Context, nsName namespace.Name) (bool, error) {
```

Comment:
Add todo to remove entire activity after 1.26 too?

Reply:
It is tricky. If there is running delete namespace workflow when upgrade from 1.26 to future version, it will break replay if we remove this activity.